index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="flex-container">
+      <div class="logo-container">
+        <a class="logo" href="">My Logo</a>
+      </div>
+      <nav class="nav_item">
+        <ion-icon class="menu" name="menu-sharp"></ion-icon>
+      </nav>
+    </header>
```

Comment:
Kindly increase the width of your hamburger menu icon to meet the required design template, as this looks too small and may not meet a user requirement 😄 

Figma design :
![image](https://github.com/Nelofarzabi/My_Portfolio/assets/20537712/66d21a1a-a4dd-4d89-92ea-78ff1b98ad63)

Your design :
![image](https://github.com/Nelofarzabi/My_Portfolio/assets/20537712/981554e9-4694-4e10-827b-e69ea8dd8bed)